src/features/unified-share-modal/UnifiedShareModal.js

```diff
@@ -128,11 +130,33 @@ class UnifiedShareModal extends React.Component<USMProps, State> {
     };
 
     renderUSF = () => {
-        const { sharedLinkEditTagTargetingApi, sharedLinkEditTooltipTargetingApi } = this.props;
+        const {
+            onRemoveAllRestrictedCollabs,
+            // $FlowFixMe - For temporary backwards compatibility
+            onRemoveAllRestrictedExternalCollabs,
+            restrictedCollabEmails,
+            // $FlowFixMe - For temporary backwards compatibility
+            restrictedExternalCollabEmails,
+            sharedLinkEditTagTargetingApi,
+            sharedLinkEditTooltipTargetingApi,
+        } = this.props;
         const { isFetching, sharedLinkLoaded, shouldRenderFTUXTooltip } = this.state;
+
+        // TODO
+        // For temporary backwards compatibility. Remove legacy onRemoveAllRestrictedExternalCollabs
+        // and restrictedExternalCollabEmails props once EUA has been updated not to pass them
+        const onRemoveAllRestrictedCollabsValue = isNil(onRemoveAllRestrictedExternalCollabs)
```

Comment:
I'm planning to remove this as soon as EUA is updated, but I also want to be able to update BUIE independently of the EUA changes in order to avoid potential sync issues.